setup.py

```diff
@@ -22,7 +22,7 @@
     description="All-in-one speech toolkit in pure Python and Pytorch",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    author="Mirco Ravanelli & Others",
+    author="Mirco Ravanelli, Titouan Parcollet & Others",
```

Comment:
Shouldn't this be "Mirco Ravanelli, Titouan Parcollet, Adel Moumen & Others" ? 😂

Reply:
Not yet 👀 One day it will be "Adel Moumen, Peter Plantinga, & Others" dw 👀